src/ngx_http_lua_ssl_certby.c

```diff
@@ -1102,6 +1147,40 @@ ngx_http_lua_ffi_parse_pem_priv_key(const u_char *pem, size_t pem_len,
 }
 
 
+void *
+ngx_http_lua_ffi_parse_pem_priv_key_with_password(const u_char *pem,
+    size_t pem_len, const u_char *pwd, size_t pwd_len, char **err)
+{
+    BIO         *in;
+    EVP_PKEY    *pkey;
+    ngx_str_t   password;
```

Comment:
Coding style issue. Please use the `ngx-releng` script to check your code. Thank you.